### DISCUSSION TITLE
Tasmota v9.5: Tuya Shutter issue

### DISCUSSION THREAD

**Discussion by mikip4** [arendst/Tasmota#20569] [Q&A]
Hello, problem is en version of tasmota.
Install 9.5.0

then

put your IP address:
IPAddress1 192.168.0.xx
IPAddress2 192.168.0.1
IPAddress3 255.255.255.0
IPAddress4 8.8.8.8
restart 1

in config/ other put:
{"NAME":"Moes WS-EUR-CW","GPIO":[0,2272,0,2304,0,0,0,0,0,0,0,0,0,0],"FLAG":0,"BASE":54}

console:
SetOption80 1
ShutterMode 1 
Interlock 1,2
Interlock ON 

TuyaMCU 11,2
TuyaMCU 12,3
TuyaMCU 13,4 

Rule1 ON Power1#state=1 DO TuyaSend4 1,0 ENDON ON Power2#state=1 DO TuyaSend4 1,2 ENDON   ON Power3#state=1 DO TuyaSend4 1,1 ENDON ON Power1#state=0 DO TuyaSend4 1,1 ENDON ON Power2#state=0 DO TuyaSend4 1,1 ENDON ON Power3#state=0 DO TuyaSend4 1,1 ENDON Rule1 1

ShutterOpenDuration DURATION OPEN OR CLOSE IN SECONDS

PowerOnState 0   Relays OFF on start

Thanks a lot!!

**Comment by Jason2866** [comment]
There is no bug fixing for old Tasmota versions. If you encounter issues update to latest version.
If the issue still exists open an issue and fill out every point asked in issue template.

**Comment by stefanbode** [comment]
Agree for the approach. And additionally as I do not own a TuyaMCU. What is the problem that you encounter. I just see the config.
